commons/src/main/scala/com/lynbrookrobotics/potassium/commons/electronics/DynamicBraking.scala

```diff
@@ -0,0 +1,41 @@
+package com.lynbrookrobotics.potassium.commons.electronics
+
+import squants.electro.{ElectricPotential, Volts}
+import squants.motion.AngularVelocity
+
+object DynamicBraking {
+  // | -> short
+  // . -> 0 volts
+  private val brakingPatterns = List(
+    "||||||||||||", // strongest braking
+    ".|||||||||||",
+    ".|||||.|||||",
+    ".|||.|||.|||",
+    ".||.||.||.||",
+    ".|.|.|.||.||",
+    ".|.|.|.|.|.|",
+    "..|..|.|.|.|",
+    "..|..|..|..|",
+    "...|...|...|",
+    ".....|.....|",
+    "...........|"  // weakest braking
+  )
+    .map(_.toCharArray)
+    .map(_.map(_ == '|'))
+  private val ditherLength = brakingPatterns.map(_.length).min
+
+  def orBrake(max: ElectricPotential, free: AngularVelocity)
```

Comment:
This name is pretty confusing, it doesn't explain what the method is supposed to calculate. How about something like `dynamicBrakingOutput` or something else that explains the purpose and result of the method.